plugins/kotlin/j2k/shared/tests/testData/newJ2k/postProcessing/GuardClause.kt

```diff
@@ -1,5 +1,47 @@
 class Test {
-    fun test(s: String?) {
-        requireNotNull(s) { "s should not be null" }
+    fun testRequire(s1: String, b1: Boolean, b2: Boolean) {
+        requireNotNull(s1) { "s should not be null" }
+
+        require(b1)
+        println("never mind")
+        // comment above b2
+        require(!b2)
+
+        require(!(!b1 && b2))
+        println(1)
+        println(2)
```

Comment:
To not insert an extra newline, it seems that you should set `lineBreaksAfter` to zero of the last statement in a block `else` branch.

Reply:
Turns out the last statement inside the else block was the culprit!